README.md

```diff
@@ -1,4 +1,4 @@
-# WP Endpoints: View
+# WP Endpoints: Static
```

Comment:
Yes just wanted to mentioned on this PR not on the line. I don't think the `--save` flag exists on composer,